style.css

```diff
@@ -0,0 +1,1358 @@
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  width: 88vw;
+  margin-bottom: 10vh;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+/* About me  */
+
+.section-3 {
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
+
+.about-me-desk {
+  display: none;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+}
+
+.h31 {
+  margin-top: 10vh;
+  margin-left: 18px;
+  margin-right: 18px;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.p31-desk {
+  margin-top: 20px;
+  margin-left: 18px;
+  margin-right: 18px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.get-myresume {
+  display: flex;
+  padding: 12px;
+  background: #ff6b00;
+  text-decoration: none;
+  margin-top: 25px;
+  margin-left: 18px;
+  width: 143px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  letter-spacing: 0.03em;
+}
+
+.abt-me-image {
+  align-self: flex-start;
+  width: 30vw;
+  height: 30vh;
+  position: relative;
+  left: 69vw;
+  margin-top: 0;
+}
+
+hr.line-3 {
+  display: flex;
+  margin-top: 5vh;
+  background-color: #e5e5e5;
+  height: 1px;
+  width: 90vw;
+  margin-left: 4vw;
+}
+
+ul.lists {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 12vw;
+  margin-top: 40px;
+  width: 70vw;
+}
+
+.Lan {
+  font-family: "InaiMathi", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: rgb(247, 247, 247);
+}
+
+.Lan::before {
+  content: "\25C6";
+  color: #ff6b00;
+  display: inline-block;
+  width: 1.3em;
+  margin-left: -1em;
+}
+
+.lan-list {
+  display: flex;
+  flex-direction: column;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: rgb(247, 240, 240);
+}
+
+.Frame {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: rgb(247, 247, 247);
+}
+
+.Frame::before {
+  content: "\25A0";
+  font-size: 25px;
+  color: #ff6b00;
+  display: inline-block;
+  width: 10px;
+  width: 1em;
+  margin-left: -1em;
+}
+
+.Frame-list {
+  display: flex;
+  flex-direction: column;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  align-items: center;
+  text-align: center;
+  color: rgb(247, 240, 240);
+}
+
+.Sk {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  align-items: center;
+  color: rgb(247, 247, 247);
+}
```

Comment:
- [ ] From your design I can see that `Skill` is not centered as it should be to match the given design guideline. Kindly fix this issue.
- ***Your Design***
- ![image](https://user-images.githubusercontent.com/60083437/234992032-1bd1083d-a83f-468e-b619-326c064f9a14.png)

- [ ] From the given design I can see there are some gaps/margin/padding under the `Skill` section. But in your design there is no space below `Skill`.
- ***Given Design***
- ![image](https://user-images.githubusercontent.com/60083437/234992470-99e8c155-8eda-491f-9838-7458930b223f.png)
- ***Your Design***
- ![image](https://user-images.githubusercontent.com/60083437/234992592-7aa86e8d-1080-42bc-ae08-d48a309621e7.png)